src/test/java/teammates/test/cases/action/InstructorCourseEnrollSaveActionTest.java

```diff
@@ -222,15 +222,12 @@ public void testExecuteAndPostProcess() throws Exception {
         
         ______TS("Boundary test for size limit per enrollment");
         
-        //TODO: sync this var with SIZE_LIMIT_PER_ENROLLMENT defined in StudentsLogic, by putting it in config or Const class
-        int sizeLimitBoundary = 150;
-        
         //can enroll, if within the size limit
         StringBuilder enrollStringBuilder = new StringBuilder(200);
         enrollStringBuilder.append("Section\tTeam\tName\tEmail");
-        for (int i = 0; i < sizeLimitBoundary; i++) {
+        for (int i = 0; i < Const.SIZE_LIMIT_PER_ENROLLMENT; i++) {
             enrollStringBuilder.append(Const.EOL).append("section" + i + "\tteam" + i + "\tname" + i
-                                                         + "\temail" + i + "@nonexistemail.nonexist");
+                    + "\temail" + i + "@nonexistemail.nonexist");
```

Comment:
For some reason git isn't registering that I changed that.  I am researching why this is happening.